components/style/values/specified/box.rs

```diff
@@ -37,6 +37,16 @@ fn moz_box_display_values_enabled(context: &ParserContext) -> bool {
         }
 }
 
+#[cfg(any(feature = "gecko", feature = "servo-layout-2013"))]
+fn unimplemented_in_servo_2020(_context: &ParserContext) -> bool {
+    true
+}
+
+#[cfg(feature = "servo-layout-2020")]
+fn unimplemented_in_servo_2020(_context: &ParserContext) -> bool {
+    servo_config::prefs::pref_map().get("layout.2020.unimplemented").as_bool().unwrap_or(false)
```

Comment:
This logic reads backwards... You only parse it if it's unimplemented?

I assume that this is what you want and that that pref would always be false, but why making it a pref at all? You could just stash `false` here and use `parse(condition = "not_servo_2020")` or such, which would read a bit nicer?

Reply:
Oh yeah it’s totally backward :] Yes making it always false based on the build-time flag also works, but since there’s a pref for properties this felt more consistent.

Reply:
Wait no, it’s not backward. The layout logic for this value is unimplemented. We only parse the keyword if the “enable unimplemented syntax” pref is on. I’ve renamed to `parse_unimplemented_in_servo_2020` to hopefully clarify.

This pref is basically never intended to be on, but prefs are what already had support in Mako template for disabling parsing of some properties while still generating a Rust type for their values.

Reply:
Right, my point is that it reads backwards. I understand what it's doing, it just reads "parse if unimplemented_in_servo_2020", which is confusing, I think. Prefixing it with `parse_` makes it a bit less confusing I guess.